File: cowork-common/src/main/golang/color.go
package common

import "fmt"

const (
	red    = 31
	green  = 32
	yellow = 33
	blue   = 36
	grey   = 38
)

func color(i int, str string) string { return fmt.Sprintf("\x1b[%dm%s\x1b[0m", i, str) }

// Red returns string with red color.
func Red(str string) string { return color(red, str) }

// Green returns string with green color.
func Green(str string) string { return color(green, str) }

// Yellow returns string with yellow color.
func Yellow(str string) string { return color(yellow, str) }

// Blue returns string with blue color.
func Blue(str string) string { return color(blue, str) }

// Grey returns string with grey color.
func Grey(str string) string { return color(grey, str) }

File: cowork-common/src/color.rs
const RED: &str = "31";
const GREEN: &str = "32";
const YELLOW: &str = "33";
const BLUE: &str = "36";
const GREY: &str = "38";

fn color(i: &str, str: &str) -> String { format!("\x1b[{}m{}\x1b[0m", i, str) }

pub fn red(str: &str) -> String { color(RED, str) }

pub fn green(str: &str) -> String { color(GREEN, str) }

pub fn yellow(str: &str) -> String { color(YELLOW, str) }

pub fn blue(str: &str) -> String { color(BLUE, str) }

pub fn grey(str: &str) -> String { color(GREY, str) }